lib/sql-string.js

```diff
@@ -119,9 +119,9 @@ SqlString.arrayToList = function(array, timeZone, dialect, field) {
       // Need to translate DATETIME to TIMESTAMP WITH TIME ZONE for Postgres here
       // There has to be a better solution than this
       if (dialect === "postgres") {
-        ret += '::' + field.type.replace(/\(\d+\)/g, '').replace(/DATETIME/, 'TIMESTAMP WITH TIME ZONE');
+        ret += '::' + field.type.toSql().replace(/\(\d+\)/g, '').replace(/DATETIME/, 'TIMESTAMP WITH TIME ZONE');
```

Comment:
Good question, i probably haven't removed the weird replaces everywhere. But no it could probably be killed since the only difference is the TIMESTAMP replacement right?